unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SocialBarPrefabs/SocialBarV1/Prefabs/ChannelMembersHUD.prefab

```diff
@@ -2236,7 +2236,7 @@ PrefabInstance:
     - target: {fileID: 33180226908608850, guid: 15120eceeeb9c45ee9d520a23f50872d,
         type: 3}
       propertyPath: m_IsActive
-      value: 1
+      value: 0
       objectReference: {fileID: 0}
```

Comment:
also, I saw a lot of deactivating components in prefabs. Please check that all of them were intentional.

Reply:
Oh... damn it. They were prefabs commited by mistake. I don't know why they become modified each time we save something in Unity editor.

Reply:
Ok, I detected 4 of these prefabs commited by mistake:
```
ChannelMembersHUD
ChatHUD
NearbyMembersHUD
UserContextMenuPanel
```
I have already reverted the changes on them here: https://github.com/decentraland/unity-renderer/pull/5318/commits/1e8f0d4b8d0394c9145111f79d4e169e4486067f